examples/new-state/src/lib.rs

```diff
@@ -215,7 +215,7 @@ mod tests {
             Some(Ok(expected_token_count))
         );
 
-        let mut a_set_iter = state_reloaded.another_struct.a_set.iter();
+        let mut a_set_iter = state_reloaded.another_struct.a_set.iter().unwrap();
```

Comment:
Use `expect` with a meaningful explanation so that if it fails a useful message is displayed.